### PR TITLE
Loaders: Replace inflate with fflate.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: see https://github.com/mrdoob/three.js/issues/20941#issuecomment-752032414

**Description**

This PR replaces the usage of `inflate.js` with `fflate.js` in `FBXLoader`, `VTKLoader` and `EXRLoader`.

/cc @sciecode, @looeee

### CODE REVIEW

**Review by 101arrowz — APPROVED** [review]
Mainly to assure users that Three.js is fully FOSS, it might be useful to add doc comments to the minified files for `fflate`:

`fflate.js`:
```js
/*!
fflate - fast JavaScript compression/decompression
<https://101arrowz.github.io/fflate>
Licensed under MIT. https://github.com/101arrowz/fflate/blob/master/LICENSE
Subset included: synchronous unzip, gunzip, and unzlib
*/
```
`fflate-deflate.js` (if this is added down the line):
```js
/*!
fflate - fast JavaScript compression/decompression
<https://101arrowz.github.io/fflate>
Licensed under MIT. https://github.com/101arrowz/fflate/blob/master/LICENSE
Subset included: synchronous deflate
*/
```

**Comment by sciecode** [comment]
Tested all internal `EXRLoader` usages of `fflate.js`, everything looks in order.

**Comment by Mugen87** [comment]
> Mainly to assure users that Three.js is fully FOSS, it might be useful to add doc comments to the minified files for fflate:

Done for `fflate.min.js` and `fflate.module.min.js`. `fflate-deflate` is not yet part of the repo.

File: examples/js/loaders/EXRLoader.js [hunk 1]
```diff
@@ -1614,8 +1611,9 @@ THREE.EXRLoader.prototype = Object.assign( Object.create( THREE.DataTextureLoade
 			if ( dwaHeader.rleRawSize > 0 ) {
 
 				var compressed = info.array.slice( inOffset.value, inOffset.value + dwaHeader.rleCompressedSize );
-				var inflate = new Inflate( compressed, { resize: true, verify: true } ); // eslint-disable-line no-undef
-				var rleBuffer = decodeRunLength( inflate.decompress().buffer );
+				var data = fflate.unzlibSync( compressed ); // eslint-disable-line no-undef
+				var acBuffer = new Uint16Array( data.buffer );
```
Comment (sciecode) [inline examples/js/loaders/EXRLoader.js:1615]: minor thing, `acBuffer` not used here, can remove and switch `rleBuffer` to:
```js
var rleBuffer = decodeRunLength( data.buffer ); ; //line 1615
```

**Review by sciecode — CHANGES_REQUESTED** [review]

File: examples/js/loaders/EXRLoader.js (hunk 1 above)
Comment (sciecode) [reply examples/js/loaders/EXRLoader.js:1615]: not as minor as I though, causes conflict with upper variable name and generates incorrect data.

[current](https://user-images.githubusercontent.com/3927951/103316988-f412d200-4a08-11eb-8c73-80a62ef66c26.png) - [change](https://user-images.githubusercontent.com/3927951/103317025-1c023580-4a09-11eb-981f-96230c18e635.png)

Comment (Mugen87) [reply examples/js/loaders/EXRLoader.js]: Fixed!

**Review by sciecode — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**